### ISSUE TITLE
Support different meta og:img per site

### ISSUE THREAD

**Issue by mcglincy** [exlibrisrpg/exlibris#764]
This may be: add a field to SystemResource and use it in meta_tags_component.rb.

I'll let John verify before I create a PR.

**Comment by jdbann** [comment]
Yeah, that could work. You'd need to add something like `has_one_attached :share_image` to the `System` model as well.

Also to make sure that the various sites can cache this effectively, we should use `rails_storage_proxy_url(Current.system&.share_image)` to generate the `image_src` attribute in `MetaTagsComponent`.

Without it, Rails generates signed URLs which load directly from the S3 bucket. That's fine for users and saves on server load, but signing it means that the URL is different each time and would keep invalidating the caches for the social platforms.
